screens/Category.js

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import {
+  StyleSheet,
+  View,
+  ImageBackground,
+  TouchableOpacity,
+  Text,
+  ScrollView,
+} from "react-native";
+import MaterialIcons from "react-native-vector-icons/MaterialIcons";
+
+import bg from "../assets/bg.png";
+
+export default function Category({ navigation }) {
+  const styles = StyleSheet.create({
+    container: {
+      flex: 1,
+      alignItems: "center",
+    },
+    back: {
+      flex: 1,
+    },
+    arrow: {
+      position: "absolute",
+      left: 20,
+      top: 40,
+    },
+    title: {
+      fontSize: 42,
+      color: "#2D2D2D",
+      marginTop: 22,
+    },
+    button: {
+      width: 320,
+      height: 50,
+      backgroundColor: "#FAFAFA",
+      borderRadius: 18,
+      justifyContent: "center",
+      alignItems: "center",
+      margin: 10,
+      borderWidth: 1,
+      borderColor: "#DBDBDB",
+      shadowColor: "#000",
+      shadowOffset: {
+        width: 0,
+        height: 5,
+      },
+      shadowOpacity: 0.25,
+      shadowRadius: 3.84,
+
+      elevation: 5,
+    },
+    list: {
+      margin: 10,
+      marginTop: "18%",
+      height: "52%",
+    },
+    name: {
+      color: "#2D2D2D",
+      fontSize: 18,
+    },
+  });
+
+  const [category, SetCategory] = React.useState([]);
+
+  const getMovieRequest = async () => {
+    const url = `https://opentdb.com/api_category.php`;
+
+    const response = await fetch(url);
+    const responseJson = await response.json();
+
+    if (responseJson.trivia_categories) {
+      SetCategory(responseJson.trivia_categories);
+    }
+  };
+
+  React.useEffect(() => {
+    getMovieRequest();
+  }, []);
```

Comment:
I think the `getMovieRequest` should be included in the dependency array, isn't eslint complaining about this one?

Reply:
eslint shows no errors.  I think you don't need to enter into the dependency array, you only need to download all available categories once.